jquery.passwd_field.js

```diff
@@ -0,0 +1,78 @@
+/*
+ * Password Visibility Toggle JQuery Plugin
+ *
+ * Adds password visibility toggle to a password field,
+ * by creating a another (text) input field. And switches
+ * between the password and the text input fields.
+ */
+(function ($) {
+
+    $.fn.passwordToggle = function () {
+
+        var docFrag = document.createDocumentFragment();
+        var random = [Math.floor(Math.random() * 9999)];
```

Comment:
If you triggered it separately on different fields, the counter would conflict. Or I have to add the counter on jquery level. That's possible.